arielperez-app/src/main/java/edu/arielperez/advancedjava/application/StockQuoteApplication.java

```diff
@@ -0,0 +1,56 @@
+package edu.arielperez.advancedjava.application;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.List;
+
+import edu.arielperez.advancedjava.model.StockQuote;
+import edu.arielperez.advancedjava.service.StockService;
+import edu.arielperez.advancedjava.service.StockServiceFactory;
+
+public class StockQuoteApplication {
+
+	public static void main(String[] args) {
+
+		String symbol = "";
+		Calendar fromDate = Calendar.getInstance();
+		Calendar toDate = Calendar.getInstance();
+
+		symbol = args[0];
+		
+		try {
+			fromDate.setTime(new SimpleDateFormat("MM/dd/yyyy").parse(args[1]));
+		} catch (ParseException e1) {
+			// TODO Auto-generated catch block
```

Comment:
No to do items in 'done' (ready to grade, review) code. 
Also, use proper exception handling always. 
This is not exception handling (it does't resolve or handle the exception) it just logs the exception. This is called dropping the execution and is not good. 
Read chapter 9 in Effective Java on exceptions.